Makefile

```diff
@@ -192,35 +179,27 @@ check-local-nix-derivations: regen-local-nix-derivations
 services: init install
 	$(MAKE) -C services/nginz
 
-# formats all Haskell files (which don't contain CPP)
+# formats everything according to treefmt rules
+# this may take a while (5 minutes) on first run but should be instant on 
+# any subsequent run
```

Comment:
```suggestion
# any subsequent run except after you have changed files.
```

...  right?  this is nice, though.

Reply:
yeah, depends on what instant means to you ;D